src/content/ui-api/observability/prometheus/disable-monitoring/index.md

```diff
@@ -0,0 +1,78 @@
+---
+linkTitle: Disable monitoring
+title: Disable Monitoring
+description: This article explains how to turn off all monitoring for an entire workload cluster.
+weight: 50
+menu:
+  main:
+    identifier: uiapi-observability-prometheus-disable
+    parent: uiapi-observability-prometheus
+user_questions:
+  - How can I disable monitoring on my cluster?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2022-11-21
+---
+
+In this article you will learn how you can disable monitoring for your cluster.
+
+## Introduction to monitoring
+
+Each cluster created on the Giant Swarm platform benefits from our monitoring which allow us to provide you with 24/7 support to ensure best quality of service.
+
+Each cluster is monitored by a dedicated Prometheus instance.
+This comes by default and storage is reserved for data retention, storage size can be adjusted via [Prometheus Volume Sizing](https://docs.giantswarm.io/ui-api/observability/prometheus/volume-size/) feature.
+
+## Why would I like to disable monitoring?
+
+You might create clusters where you do not need support from our operations team, for instance because you might just want to try something new for a while, or you are running a playground cluster where you know what you are doing.
+You might as well have storage constraints and therefore would like to turn monitoring off.
+
+## What does disabling monitoring involve?
+
+When you turn off monitoring this has several implication:
+
+- Giant Swarm will not actively provide support on this cluster, as we will not be receiving alerts for this cluster.
+- Giant Swarm will not be able to provide detailed investigation, as we will not have monitoring data over time to read from.
+
+But nevertheless we are still here to help and support you in case you need it.
+
+## How to disable monitoring?
+
+To disable monitoring for a cluster you need to label the Cluster CR with:
+
+```yaml
+giantswarm.io/monitoring: "false"
+```
+
+You can achieve this by running the following command :
+
+```nohighlight
+$ kubectl label cl myclustername giantswarm.io/monitoring=false --overwrite
```

Comment:
Reagrding `kubectl label cl`. In docs we normally go for explicit rather than short and implicit. I would recommend to use the entire resource name instead of `cl`. Even `cluster` might be ambiguous.

Reply:
Yes `cluster` is ambiguous and do not point to the correct resource, so the other solution is to use `clusters.cluster.x-k8s.io` does this sounds better to you ?

Reply:
yeah, +1 on being super unambiguous

Reply:
Yep, let's use `clusters.cluster.x-k8s.io`. That's also what we do in CLI guides in happa btw.

Reply:
Fixed :heavy_check_mark: